From 147393b185bef62925497f42069994831c590141 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 8 Jun 2018 10:25:47 +0800
Subject: [PATCH] arm64: rk3326_linux_defconfig: add the missing RK8xx
 configigure

This patch enables the gpio and powerkey configure for Rockchip
RK8xx PMIC.

Change-Id: Iab4f9303922387ae6e412f06aa098cdf53331e87
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rk3326_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rk3326_linux_defconfig b/arch/arm64/configs/rk3326_linux_defconfig
index 47c2bfe4cafd..533f9fb884d7 100644
--- a/arch/arm64/configs/rk3326_linux_defconfig
+++ b/arch/arm64/configs/rk3326_linux_defconfig
@@ -160,6 +160,7 @@ CONFIG_TOUCHSCREEN_GT1X=y
 CONFIG_ROCKCHIP_REMOTECTL=y
 CONFIG_ROCKCHIP_REMOTECTL_PWM=y
 CONFIG_INPUT_MISC=y
+CONFIG_INPUT_RK8XX_PWRKEY=y
 CONFIG_INPUT_UINPUT=y
 CONFIG_INPUT_GPIO=y
 # CONFIG_SERIO is not set
@@ -188,6 +189,7 @@ CONFIG_PTP_1588_CLOCK=y
 CONFIG_DEBUG_GPIO=y
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_GENERIC_PLATFORM=y
+CONFIG_GPIO_RK8XX=y
 CONFIG_BATTERY_SBS=y
 CONFIG_CHARGER_GPIO=y
 CONFIG_CHARGER_BQ24735=y
-- 
2.35.3

